### PR TITLE
refactored config reading

### LINKED ISSUE #203: Missing  PROCESSING_ section in configuration.go
https://github.com/RedHatInsights/insights-results-aggregator/blob/master/configuration.go#L101

### PR DESCRIPTION
**Pull request by Sergey1011010**
# Description

Fixes #203

## Type of change

Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)
- Refactor (refactoring code, removing useless files)

## Testing steps

`make before_commit`

### CODE REVIEW

File: aggregator_test.go [hunk 1]
```diff
@@ -36,42 +35,7 @@ func TestStartStorageConnection(t *testing.T) {
...
-			pg_db_name = ""
-			pg_params = ""
-		`)
-
-		viper.SetConfigType("toml")
-		err := viper.ReadConfig(bytes.NewBuffer(config))
-		if err != nil {
-			t.Fatal(err)
-		}
+		os.Clearenv()
```
Comment (tisnik) [inline aggregator_test.go]: ah interesting trick :)

File: config.toml [hunk 2]
```diff
@@ -13,11 +13,6 @@ debug = true
 [processing]
 org_whitelist = "org_whitelist.csv"
```
Comment (tisnik) [inline config.toml]: were metric removed on purpose?

File: configuration.go [hunk 3]
```diff
@@ -148,65 +174,14 @@ func loadWhitelistFromCSV(r io.Reader) (mapset.Set, error) {
 
 		orgID, err := strconv.Atoi(line[0]) // single record per line
```
Comment (tisnik) [inline configuration.go]: just a though: orgID might be unsigned int, not signed one ;)

File: config.toml (hunk 2 above)
Comment (Sergey1011010) [reply config.toml]: They are not used anywhere

File: configuration.go (hunk 3 above)
Comment (Sergey1011010) [reply configuration.go]: fixed

**Pull request merged**